src/main/java/mcnutty/music/get/ProcessQueue.java

```diff
@@ -11,16 +11,29 @@
     public ArrayList<QueueItem> bucket_played;
     public ArrayList<QueueItem> bucket_youtube;
 
+    private static final int max_buckets = 4;
+
```

Comment:
NM, submitted a request for better config.ini: https://github.com/mcnutty26/music-get/issues/28